pages/1-discovery-research.md

```diff
@@ -6,12 +6,13 @@ The first stage of any project is to do research to discover problems that need
 
 These design methods can help guide your research with both stakeholders and possible end users:
 
->- [Contextual inquiry](https://methods.18f.gov/contextual-inquiry/)[](https://methods.18f.gov/bodystorming/) 
```

Comment:
Not sure why the `>` was before this list item. I noticed it in other lists as well.